src/headphones2/postprocess/component_base.py

```diff
@@ -0,0 +1,13 @@
+import abc
+
+
+class PostProcessorComponentBase(object):
```

Comment:
`ComponentBase` is redundant since it is ABCMeta and any class is a componenet...